src/main/java/pl/wedding/management/service/user/model/User.java

```diff
@@ -0,0 +1,28 @@
+package pl.wedding.management.service.user.model;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import java.util.Objects;
+
+@Builder
+@Getter
+public class User {
+
+    private final Integer userId;
+    private final EMail email;
+    private final char[] password;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        User user = (User) o;
+        return Objects.equals(email, user.email);
+    }
```

Comment:
Poprawiłbym porównanie kont - chodzi mi o sytuację, w której User z adresem mailowym "adresemail@gmail.com" będzie tym samym userem co ten z adresem "adres.email@gmail.com"

Reply:
przecież to są dwa różne maile - dwie różne osoby. już teraz ciężko jest dostać taki sam email dajmy na to na gmailu. 
nie mogę zrobić sobie sienkiewiczmaciej2@gmail.com bo taki jest już zajęty. dlaczego sienkiewiczmaciej@gmail.com miałby być tym samym użytkownikiem niż sienkiewicz.maciej@gmail.com?

Reply:
Bartek wytłumaczył mi, że kropki dla gmaila nie wpływają na interpretację adresu. Wcześniej o tym nie wiedziałem. 
Zgadzam się z jego uwagą: być może będzie warto rozważyć taką sytuację i zaimplementować to podejście w przyszłości.

Reply:
It's only gmail standard, which differs from specification. Good feature, but only google specfic.